common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -149,10 +150,20 @@ public void setExternalLogger(final ILoggerCallback externalLogger) {
      * @return String The concatenation of thread_name and correlation_id to serve as the required metadata in the log lines.
      */
     public static String getDiagnosticContextMetadata() {
+        String threadName = DiagnosticContext.getRequestContext().get(DiagnosticContext.THREAD_NAME);
+        String correlationId = DiagnosticContext.getRequestContext().get(DiagnosticContext.CORRELATION_ID);
+
+        if(StringUtil.isEmpty(threadName)){
```

Comment:
```suggestion
        if (StringUtil.isEmpty(threadName)){
```

(Consider using `ctrl` + `alt` + `l` to auto format code for consistency)

Reply:
addressed, thanks :)